index.js

```diff
@@ -66,44 +84,17 @@ class KoaWebMonetization {
     this.buckets.set(id, balance - price)
   }
 
-  paid ({ price, awaitBalance = false }) {
-    return async (ctx, next) => {
-      const id = ctx.params.id
-      if (!id) {
-        return ctx.throw(400, 'ctx.params.id must be defined')
-      }
-
-      const _price = (typeof price === 'function')
-        ? Number(price(ctx))
-        : Number(price)
-
-      if (awaitBalance) {
-        await this.awaitBalance(id, _price)
-      }
-
-      try {
-        this.spend(id, _price)
-        return next()
-      } catch (e) {
-        return ctx.throw(402, e.message)
-      }
-    }
-  }
-
-  receiver () {
-    return async ctx => {
-      await this.connect()
-
-      if (ctx.get('Accept').indexOf('application/spsp+json') === -1) {
-        return ctx.throw(404)
-      }
-
+  async receive (ctx, next) {
```

Comment:
we should probably not have this documented in the API if it's being done automatically as a part of `mount`. It's more of an internal method now.